<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>libetonyek.git - libetonyek</title>
<meta name='generator' content='cgit v0.10.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/libetonyek.git/atom/0001-fix-test-on-i386.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 4 && tokens[4] != '') {
    repo = tokens[4].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='libetonyek.git' href='/cgit/libetonyek.git/'>libetonyek.git</a></td><td class='form'><form method='get' action=''>
<input type='hidden' name='id' value='19e5c4a429477c51d2bdacbe9b140e7926f242e5'/><select name='h' onchange='this.form.submit();'>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>libetonyek</td><td class='sub right'>Jon Ciesla</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/libetonyek.git/'>summary</a><a href='/cgit/libetonyek.git/refs/?id=19e5c4a429477c51d2bdacbe9b140e7926f242e5'>refs</a><a href='/cgit/libetonyek.git/log/0001-fix-test-on-i386.patch'>log</a><a class='active' href='/cgit/libetonyek.git/tree/0001-fix-test-on-i386.patch?id=19e5c4a429477c51d2bdacbe9b140e7926f242e5'>tree</a><a href='/cgit/libetonyek.git/commit/0001-fix-test-on-i386.patch?id=19e5c4a429477c51d2bdacbe9b140e7926f242e5'>commit</a><a href='/cgit/libetonyek.git/diff/0001-fix-test-on-i386.patch?id=19e5c4a429477c51d2bdacbe9b140e7926f242e5'>diff</a><a href='/cgit/libetonyek.git/stats/0001-fix-test-on-i386.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/libetonyek.git/log/0001-fix-test-on-i386.patch'>
<input type='hidden' name='id' value='19e5c4a429477c51d2bdacbe9b140e7926f242e5'/><select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/libetonyek.git/tree/?id=19e5c4a429477c51d2bdacbe9b140e7926f242e5'>root</a>/<a href='/cgit/libetonyek.git/tree/0001-fix-test-on-i386.patch?id=19e5c4a429477c51d2bdacbe9b140e7926f242e5'>0001-fix-test-on-i386.patch</a></div><div class='content'>blob: f3a7dc36e1edf69c80fabec9dee549aa9d6126dd (<a href='/cgit/libetonyek.git/plain/0001-fix-test-on-i386.patch?id=19e5c4a429477c51d2bdacbe9b140e7926f242e5'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
</pre></td>
<td class='lines'><pre><code>From 5bb538c7c9984d2f68b81eb182ca9bf4c12593a9 Mon Sep 17 00:00:00 2001
From: David Tardon &lt;dtardon@redhat.com&gt;
Date: Wed, 20 May 2015 16:44:25 +0200
Subject: [PATCH] fix test on i386

Change-Id: I54379bec41f724d5c3ff8bf9e559c5bda7a9c57d
---
 src/test/IWORKTransformationTest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/test/IWORKTransformationTest.cpp b/src/test/IWORKTransformationTest.cpp
index b44d52f..77f0b27 100644
--- a/src/test/IWORKTransformationTest.cpp
+++ b/src/test/IWORKTransformationTest.cpp
@@ -209,7 +209,7 @@ void IWORKTransformationTest::testIdentities()
   CPPUNIT_ASSERT(origin(100, 50) == translate(-50, -25));
   CPPUNIT_ASSERT((flip(true, false) * flip(false, true)) == flip(true, true));
   CPPUNIT_ASSERT((flip(false, true) * flip(true, false)) == flip(true, true));
-  CPPUNIT_ASSERT((rotate(etonyek_half_pi) * rotate(etonyek_third_pi)) == (rotate(etonyek_third_pi) * rotate(etonyek_half_pi)));
+  CPPUNIT_ASSERT(approxEqual(rotate(etonyek_half_pi) * rotate(etonyek_third_pi), rotate(etonyek_third_pi) * rotate(etonyek_half_pi)));
   CPPUNIT_ASSERT(scale(-1, -1) == flip(true, true));
   CPPUNIT_ASSERT((translate(80, 40) * translate(10, 20)) == (translate(10, 20) * translate(80, 40)));
   CPPUNIT_ASSERT((scale(2, 2) * translate(1, 2) == (translate(2, 4) * scale(2, 2))));
-- 
2.4.0

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.2 at 2015-05-24 10:23:05 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
